deps/unwind.mk

```diff
@@ -30,10 +30,6 @@ endif
 
 define LIBUNWIND_INSTALL
```

Comment:
@vtjnash Not sure I fully understand. But would be best if you make the commits.